Makefile

```diff
@@ -674,6 +679,14 @@ endif
 	$(INSTALL_FOLDER) $(DOCDIR)
 	cp README.md doc/*.md $(DOCDIR)
 	$(SED) -i  's!\./doc/!./!' $(DOCDIR)/README.md
+ifndef SKIP_CLKMGR
+	cp -a $(LIB_D)/$(CLKMGR_NAME)*.so* $(DLIBDIR)
```

Comment:
shd we force this cp  ?

In case there is already a current so in the path. 

cp -af ?